README.md

```diff
@@ -4,42 +4,42 @@ wpantund, Userspace WPAN Network Daemon
 `wpantund` is a user-space network interface driver/daemon that
 provides a native IPv6 network interface to a low-power wireless
 **Network Co-Processor** (or *NCP*). It was written and developed by
-Nest Labs to make supporting [Thread](http://threadgroup.org)
-connectivity on unix-like operating systems more straightforward.
+Nest to support [Thread](http://threadgroup.org)
+connectivity on uUnix-like operating systems and make the support more straightforward.
```

Comment:
* I'm no English major, but this new sentence structure feels awkward. Perhaps it would be worthwhile to entirely reword the sentence if my original wording wasn't any better.
* Typo: `uUnix`
* The right margin should be cleaned up a bit.